docs/03-tutorials/00-observability/obsv-01-observe-application-metrics.md

```diff
@@ -97,3 +97,19 @@ Follow these steps to redirect the metrics:
    The Prometheus UI shows a new value every 10 seconds upon refreshing the page.
 
 4. If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
+
+## Limitations
+
+The configured memory limits of the Prometheus and Prometheus-Istio instances limit the number of time series samples that can be ingested. The amount of generated time series in a Kyma cluster is depending on the following factors:
+
+* Number of Pods in the cluster
+* Number of Nodes in the cluster
+* Amount of exported (custom) metrics
+* Label cardinality of metrics
+* Number of buckets for histogram metrics
+* Frequency of Pod recreation
+* Topology of the Istio service mesh
+
+The default resource configuration of the monitoring component in the production profile is sufficient to serve 800K time series in the Prometheus Pod and 400K time series with a retention time of 30 days in the Prometheus-Istio Pod. The evaluation profile has lower limits.
```

Comment:
```suggestion
The default resource configuration of the monitoring component in the production profile is sufficient to serve 800K time series in the Prometheus Pod, and 400K time series with a retention time of 30 days in the Prometheus-Istio Pod. The evaluation profile has lower limits.
```

Maybe also add a link to 04-operation-guides/operations/02-install-kyma#choose-resource-consumption where the profiles are presented (For more information about profiles, see...). Otherwise, users might not know what that is.